src/components/select/select.js

```diff
@@ -776,6 +776,11 @@ function OptionDirective($mdButtonInkRipple, $mdUtil) {
     var optionCtrl = ctrls[0];
     var selectCtrl = ctrls[1];
 
+    if (selectCtrl.isMultiple) {
+      element.attr('md-checkbox-enabled', '');
+      element.prepend('<div class="_md-container"><div class="_md-icon"></div></div>');
```

Comment:
Instead of using a string here, define a single const that contains this "template element" and then clone it in this `prepend` call; element cloning will be significantly faster than parsing the HTML and re-creating the DOM structure every time.